ocaml/xapi-guard/src/main.ml

```diff
@@ -66,7 +71,9 @@ let recover_path = "/run/nonpersistent/varstored-guard-active.json"
 
 let store_args sockets =
   Hashtbl.fold
-    (fun path (_, (vm_uuid, gid)) acc -> {Persistent.vm_uuid; path; gid} :: acc)
+    (fun path (_, (vm_uuid, gid, typ)) acc ->
```

Comment:
Could also use `to_seq_values |> Seq.to_list`

Reply:
Needs the key too, but yes I've rewritten this using seq instead of fold, it is clearer.